hil/ext/switches/junos/config_tables/ConfigTables.py

```diff
@@ -3,5 +3,5 @@
 _YAML_ = splitext(__file__)[0] + '.yml'
 globals().update(loadyaml(_YAML_))
 
-# Reference: "https://www.juniper.net/documentation/en_US/junos-pyez/topics/task/program/junos-pyez-tables-views-external-importing.html"
-
+# Reference: "https://www.juniper.net/documentation/en_US/junos-pyez/topics/
+# task/program/junos-pyez-tables-views-external-importing.html"
```

Comment:
I don't think splitting this across two lines is a good idea. I assume you did this to satisfy pep8? I think this is a good place to just use `# noqa`

Reply:
yes I did it for satisfying pep8. Thanks for the suggestion. will do that in the next commit.